tests/test_sn_flux_est.py

```diff
@@ -125,31 +125,31 @@ def test_est_pulsar_sn():
     """
     I have tested this function various pulsar/observation combinations and compared to the estimated S/N from bestrof files. Here are the results:
 
-    1225462936, -b 1225462943 -e 1225463543 psr J0152-1637: The function estimated a S/N of 99.39 +/- 23.55. I compared this to what was estimated from the profile analyse_pulse_prof() and attained a S/N of 75.26 +/- 1.70
+    1225462936, -b 1225462943 -e 1225463543 psr J0152-1637: The function estimated a S/N of 100.92 +/- 22.48. I compared this to what was estimated from the profile analyse_pulse_prof() and attained a S/N of 75.26 +/- 1.70
 
-    1226062160, -b 1226062167 -e 1226062767 psr J2330-2005: Function estimated a S/N of 62.21 +/- 38.63. Compared this to the profile estimation and attained S/N of 51.51 +/- 1.16
+    1226062160, -b 1226062167 -e 1226062767 psr J2330-2005: Function estimated a S/N of 412.26 +/- 103.80. Compared this to the profile estimation and attained S/N of 51.51 +/- 1.16
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line too long (181 > 159 characters)](https://app.codacy.com/app/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4472703)